Remotion/Security/Core/Metadata/LocalizingMetadataConverter.cs

```diff
@@ -78,18 +78,21 @@ private LocalizedName[] GetLocalizedNames (MetadataCache cache, CultureInfo cult
 
     private LocalizedName CreateLocalizedNameFromClassInfo (SecurableClassInfo classInfo, string text)
     {
-      return new LocalizedName (classInfo.ID, classInfo.Name, classInfo.Description);
+      // TODO: Add debug notnull assertions
```

Comment:
those lazy initialzied Info-properties are not the best concept as far as null-safety is concerned. for now, we can leave the bang given (the LocalizedName does argchecks anyway and a follow-up ticket/todo to document this should be prevented via the typesystem. (e.g. builders) or by some other refactoring.